tests/__fixtures__/addresses/addresses.fixtures.mts

```diff
@@ -1,10 +1,151 @@
-import { AddressType } from '@farfetch/blackout-client';
+import {
+  type AddressSchema,
+  AddressSchemaAddressType,
+  AddressSchemaLineType,
+  AddressType,
+} from '@farfetch/blackout-client';
 
 export const addressId = '1234';
 export const addressId2 = '2222222';
 export const addressId3 = '33333333';
 export const userId = 123456;
 
+export const mockAddressSchema: AddressSchema = [
```

Comment:
There is a mock already inside `locale.fixtures.mts` (`mockGetAddressSchemaResponse`). Reuse it,please.